src/call/mod.rs

```diff
@@ -483,6 +483,14 @@ impl StreamingBase {
             Ok(Async::Ready(bytes))
         }
     }
+
+    // Cancel the call if we still have some messages or did not
+    // receive status code.
+    fn on_drop<C: ShareCallHolder>(&self, call: &mut C) {
+        if !self.read_done || self.close_f.is_some() {
```

Comment:
Should it be `is_none`?

Reply:
I think it should be `is_some`. `Some` means the call is alive.